### ISSUE TITLE
how to change dropout fraction p in dropout layer after each epoch during training?

### ISSUE THREAD

**Issue by jasonTuZq** [fchollet/keras#4597]
Hi, i want to change the dropout fraction p in dropout layer after each epoch during training, like p=p/2, but not knowing how to do.
Can anyone help me?
Thanks!

Please make sure that the boxes below are checked before you submit your issue. Thank you!

- [x] Check that you are up-to-date with the master branch of Keras. You can update with:
pip install git+git://github.com/fchollet/keras.git --upgrade --no-deps

- [x] If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps

- [x] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).
